daq/faucet_states_collector.py

```diff
@@ -15,8 +18,8 @@ def set_default(obj):
 
     def wrapped(self, *args, **kwargs):
         res = func(self, *args, **kwargs)
-        _output = json.dumps(self.system_states, default=set_default)
-        #print(_output)
+        with self.lock:
+            LOGGER.debug(json.dumps(self.system_states, default=set_default))
```

Comment:
please keep this suppressed (commented out) -- it's too noisy to have it logging all the time

Reply:
Not even debug?

Reply:
Can set it to NOTSET